trin-core/src/portalnet/types/content_keys.rs

```diff
@@ -435,10 +472,17 @@ mod test {
         } else {
             panic!("Assertions not tested. Check if proper decoding occurred")
         }
+
+        // TESTING CONTENT-ID DERIVATION
+        let actual_id = test_key.to_content_id().unwrap().to_string();
+        let expected_id = String::from(
```

Comment:
*nitpick* - In these conversions throughout the tests, I'd go with `"804....".to_string()` rather than `String::from("804...")` just to be similar with the previous step - but, that's just me, I could be easily convinced if you prefer the way it is